src/rgw/rgw_rados.cc

```diff
@@ -6459,7 +6459,7 @@ static int _get_obj_iterate_cb(const rgw_raw_obj& read_obj, off_t obj_ofs,
 
 int RGWRados::flush_read_list(struct get_obj_data* d)
 {
-
+  lsubdout(g_ceph_context, rgw_datacache, 20) << "D3nDataCache: " << __func__ << "()" << dendl;
   d->d3n_datacache_lock.lock();
   list<bufferlist> l;
```

Comment:
I hate std::list--could this be a vector?

Reply:
would it be ok to postpone this refactoring to after the initial D3N merge